atom/browser/net/atom_network_delegate.cc

```diff
@@ -435,7 +435,7 @@ bool AtomNetworkDelegate::OnCancelURLRequestWithPolicyViolatingReferrerHeader(
 // https://crbug.com/704259
 bool AtomNetworkDelegate::OnCanQueueReportingReport(
     const url::Origin& origin) const {
-  return false;
+  return true;
```

Comment:
Is this something we might want to allow users to override.  E.g. `return Emit("will-queue-report")`

Idk what the surface of this is, is it just a web feature we should have enabled a while back or is it more 🤔

Reply:
> is it just a web feature we should have enabled a while back

yup. https://w3c.github.io/reporting

Reply:
I'm not sure it makes a ton of sense to expose this as something users might want to override. Maybe? But this is a synchronous API on the IO thread and it's unclear how to trigger an event for that effectively and get a response.

I think it makes sense to defer adding a feature for this until such a time as it is needed. Otherwise we'll likely build the wrong API.

Reply:
On further consideration, it probably does make sense to expose some of this stuff.